cirq-core/cirq/optimizers/clifford_decomposition.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility methods related to decompose clifford gate into circuits."""
+
+from typing import List, TYPE_CHECKING
+import functools
+
+import numpy as np
+from cirq import ops, protocols, qis, sim
+
+if TYPE_CHECKING:
+    import cirq
+
+
+def _X(q, args, operations, qubits):
```

Comment:
type and return annotations are missing in all these private functions.

Reply:
Done